python/tvm/topi/x86/dense_alter_op.py

```diff
@@ -25,13 +25,15 @@
 from ..utils import get_const_tuple
 from ..nn import dense_alter_layout
 from .utils import target_has_vnni
+from .utils import target_has_amx
 from .. import nn
 
 
-def check_vnni_applicable(x, y, allow_padding=False):
+def check_inst_applicable(x, y, allow_padding=False):
     mcpu = tvm.target.Target.current().mcpu
+    cpu_avai = target_has_vnni(mcpu) or target_has_amx(mcpu)
     return (
-        target_has_vnni(mcpu)
+        cpu_avai
```

Comment:
* ~cpu_avai~ may sound strange, ```cpu_simd``` ?

Reply:
yep, replaced `cpu_avai` to  `simd_avai` to check if vnni or amx was availiable  in this target.